apps/prairielearn/src/pages/partials/issueBadge.ejs

```diff
@@ -1,9 +1,9 @@
 <% if (count > 0) { %>
     <% if (!(typeof suppressLink != 'undefined' && suppressLink)) { %>
         <% if (typeof issueQid != 'undefined') { %>
-            <a class="badge badge-pill badge-danger" href="<%= urlPrefix %>/course_admin/issues?q=is%3Aopen+qid%3A<%= issueQid %>">
+            <a class="badge badge-pill badge-danger" aria-label="<%= count %> Open Issues" href="<%= urlPrefix %>/course_admin/issues?q=is%3Aopen+qid%3A<%= issueQid %>">
```

Comment:
You should handle pluralization here so that we don't announce "1 open issue".

There's no need to capitalize every word in these labels.

Reply:
Handling pluralization in [44b46e6](https://github.com/PrairieLearn/PrairieLearn/pull/10496/commits/44b46e6049e98c472fc0c68482ef1603a3395773).